hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/handlers/ContractCallLocalHandlerTest.java

```diff
@@ -192,12 +208,37 @@ void validateFailsIfNoContractOrTokenTest() {
         assertThatThrownBy(() -> subject.validate(context)).isInstanceOf(PreCheckException.class);
     }
 
+    @Test
+    void validateFailsIfGasIsLessThanIntrinsic() {
+        // given
+        given(context.query()).willReturn(query);
+        given(query.contractCallLocalOrThrow()).willReturn(contractCallLocalQuery);
+        given(contractCallLocalQuery.gas()).willReturn(INTRINSIC_GAS - 1);
```

Comment:
_This_ `INTRINSIC_GAS` is over in `TestHelpers` and is `12345`, not `21064`, is that what you intend?